test/functional/board-layout/dnd-page-object.ts

```diff
@@ -174,4 +174,11 @@ export class DndPageObject extends BasePageObject {
     const liveAnnouncements = await this.browser.execute(() => window.__liveAnnouncements ?? []);
     return liveAnnouncements;
   }
+
+  // Clicking on a drag/resize handler performs an animation (show the UAP actions, moving the board item).
+  // The pause after the click makes the functional test stable.
+  async handlerClick(selector: string) {
+    await this.click(selector);
+    await this.pause(100);
+  }
```

Comment:
While testing `test/functional/board-layout/uap-action-button-interactions.test.ts` by using click interaction without waiting, I discovered this test is flaky (sometimes elements to click are not available). To make that stable, I oriented myself on other mouse interactins (drag and drop) where we use a timeout as well.